tests/integration/annotation_import/conftest.py

```diff
@@ -11,6 +11,167 @@
 from labelbox.schema.queue_mode import QueueMode
 
 
+@pytest.fixture()
+def audio_data_row(rand_gen):
+    return {
+        "row_data":
+            "https://storage.googleapis.com/labelbox-datasets/audio-sample-data/sample-audio-1.mp3",
+        "global_key":
+            f"https://storage.googleapis.com/labelbox-datasets/audio-sample-data/sample-audio-1.mp3-{rand_gen(str)}",
+        "media_type":
+            "AUDIO",
+    }
+
+
+@pytest.fixture()
+def conversation_data_row(rand_gen):
+    return {
+        "row_data":
+            "https://storage.googleapis.com/labelbox-datasets/conversational-sample-data/sample-conversation-1.json",
+        "global_key":
+            f"https://storage.googleapis.com/labelbox-datasets/conversational-sample-data/sample-conversation-1.json-{rand_gen(str)}",
+        "media_type":
+            "CONVERSATIONAL",
+    }
+
+
+@pytest.fixture()
+def dicom_data_row(rand_gen):
+    return {
+        "row_data":
+            "https://storage.googleapis.com/labelbox-datasets/dicom-sample-data/sample-dicom-1.dcm",
+        "global_key":
+            f"https://storage.googleapis.com/labelbox-datasets/dicom-sample-data/sample-dicom-1.dcm-{rand_gen(str)}",
+        "media_type":
+            "DICOM",
+    }
+
+
+@pytest.fixture()
+def geospatial_data_row(rand_gen):
+    return {
+        "row_data": {
+            "tile_layer_url":
+                "https://s3-us-west-1.amazonaws.com/lb-tiler-layers/mexico_city/{z}/{x}/{y}.png",
+            "bounds": [[19.405662413477728, -99.21052827588443],
+                       [19.400498983095076, -99.20534818927473]],
+            "min_zoom":
+                12,
+            "max_zoom":
+                20,
+            "epsg":
+                "EPSG4326",
+        },
+        "global_key":
+            f"https://s3-us-west-1.amazonaws.com/lb-tiler-layers/mexico_city/{z}/{x}/{y}.png-{rand_gen(str)}",
+        "media_type":
+            "TMS_GEO",
+    }
+
+
+@pytest.fixture()
+def html_data_row(rand_gen):
+    return {
+        "row_data":
+            "https://storage.googleapis.com/labelbox-datasets/html_sample_data/sample_html_1.html",
+        "global_key":
+            f"https://storage.googleapis.com/labelbox-datasets/html_sample_data/sample_html_1.html-{rand_gen(str)}",
+    }
+
+
+@pytest.fixture()
+def image_data_row(rand_gen):
+    return {
+        "row_data":
+            "https://lb-test-data.s3.us-west-1.amazonaws.com/image-samples/sample-image-1.jpg",
+        "global_key":
+            f"https://lb-test-data.s3.us-west-1.amazonaws.com/image-samples/sample-image-1.jpg-{rand_gen(str)}",
+        "media_type":
+            "IMAGE",
+    }
+
+
+@pytest.fixture()
+def document_data_row(rand_gen):
```

Comment:
I have a very similar fixture in my PR, as you know, so this will be a dupe..no action I suppose, when both are merged we can get rid of one

Reply:
I've combined the two fixtures.